java/org/apache/tomcat/util/compat/Jre7Compat.java

```diff
@@ -32,6 +36,9 @@
 
 class Jre7Compat extends JreCompat {
 
+    private static final Log log = LogFactory.getLog(Jre7Compat.class);
+    private static final StringManager sm = StringManager.getManager(Jre7Compat.class);
```

Comment:
The variable should be named as stringManager instead of sm?

Reply:
No. It should be `sm` for consistency with the rest of the Tomcat code base.